src/core/selinux-access.c

```diff
@@ -164,6 +164,11 @@ static int access_init(sd_bus_error *error) {
         selinux_set_callback(SELINUX_CB_AUDIT, (union selinux_callback) { .func_audit = audit_callback });
         selinux_set_callback(SELINUX_CB_LOG, (union selinux_callback) { .func_log = log_callback });
 
+        r = audit_loginuid_from_pid(0, &manager_auid);
```

Comment:
I think it would be better to hide the manager loginuid caching inside of audit_util.c. i.e. add a helper there:

```c
uid_t audit_loginuid_cached(void) {
        static uid_t cache = UID_INVALID;
        static bool cache_valid = false;

        if (cache_valid)
                return cache;
        if (getpid_cached() == 1) 
                return UID_INVALID;   

        (void) audit_loginuid_from_pid(0, &cache);
        cache_valid = true; 
}
```

and then use that here.